### PR TITLE
Add common labels for chart objects

### PR DESCRIPTION
**Pull request by Aakcht**
This PR adds possibility to set common labels for all chart objects except for CRDs

### CODE REVIEW

File: installer/README.md [hunk 1]
```diff
@@ -116,12 +116,13 @@ The following are the list configurable parameters of Volcano Chart and their de
...
-|`custom.admission_podLabels`|Pod labels for Admission pods|`~`|
-|`custom.controller_podLabels`|Pod labels for Controller pods|`~`|
-|`custom.scheduler_podLabels`|Pod labels for Scheduler pods|`~`|
-|`custom.admission_labels`|Labels for Admission deployment and job|`~`|
-|`custom.controller_labels`|Labels for Controller deployment|`~`|
-|`custom.scheduler_labels`|Labels for Scheduler deployment|`~`|
+|`custom.admission_podLabels`|Pod labels for Admission pods|`{}`|
+|`custom.controller_podLabels`|Pod labels for Controller pods|`{}`|
+|`custom.scheduler_podLabels`|Pod labels for Scheduler pods|`{}`|
+|`custom.admission_labels`|Labels for Admission deployment and job|`{}`|
```
Comment (Monokaix) [inline installer/README.md:122]: Welcome! Why change default value to `~`?

Comment (Aakcht) [reply installer/README.md]: Hi, @Monokaix !

Sorry for this, for easirer merge I decided to set this to `{}` by default. However on a second thought it is also pretty easy to support `~` in merge, so I updated the PR to remove this change. Please let me know if anything else is needed.

**Comment by Aakcht** [comment]
Looks like the DCO check got stuck for some reason, should I try rebasing the PR?

**Comment by Monokaix** [comment]
> Looks like the DCO check got stuck for some reason, should I try rebasing the PR?

Yeah, you can have a try.

**Comment by Aakcht** [comment]
@Monokaix looks like the rebase helped, please take a look

**Comment by Aakcht** [comment]
@william-wang @Monokaix please take a look, I had to do one more rebase to fix merge conflicts

**Pull request merged**